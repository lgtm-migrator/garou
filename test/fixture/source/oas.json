{
  "security": {},
  "components": {
    "responses": {
      "201": {},
      "default": {},
      "400": {}
    }
  },
  "webhooks": {},
  "paths": {},
  "jsonSchemaDialect": "",
  "servers": [
    {
      "variables": {
        "description": "",
        "default": "",
        "enum": ""
      },
      "description": "",
      "url": ""
    }
  ],
  "info": {
    "termsOfService": "",
    "description": "",
    "version": "",
    "title": "",
    "license": {
      "name": "",
      "url": "",
      "identifier": ""
    },
    "contact": {
      "email": "",
      "url": "",
      "name": ""
    },
    "summary": ""
  },
  "openapi": "3.0.0",
  "tags": [
    {
      "description": "",
      "name": "",
      "externalDocs": {
        "description": "",
        "url": ""
      }
    }
  ],
  "externalDocs": {
    "description": "",
    "url": ""
  }
}
