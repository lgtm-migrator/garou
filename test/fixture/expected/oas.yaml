openapi: 3.0.0
info:
  version: 1
  title: test
  description: 123456789tt
